src/main/java/com/google/api/generator/gapic/protoparser/Parser.java

```diff
@@ -54,16 +54,17 @@ public GapicParserException(String errorMessage) {
               fileToGenerate);
 
       String pakkage = TypeParser.getPackage(fileDescriptor);
-      for (ServiceDescriptor serviceDescriptor : fileDescriptor.getServices()) {
-        List<Method> methods = parseMethods(serviceDescriptor, messageTypes);
-        Service service =
-            Service.builder()
-                .setName(serviceDescriptor.getName())
-                .setPakkage(pakkage)
-                .setMethods(methods)
-                .build();
-        services.add(service);
-      }
+      List<Service> parsedServices =
+          fileDescriptor.getServices().stream()
```

Comment:
Very cool. Learned that this style is very similar with Javascript map

Reply:
Great!